projects/ngx-charts-on-fhir/src/lib/fhir-chart/range-selector/range-selector.component.ts

```diff
@@ -32,6 +37,11 @@ export class RangeSelectorComponent {
     })
 
   }
+  ngOnChanges(changes: SimpleChanges){
+    const changedProp = changes["isMatGroupFocus"];
+    this._isMatGroupFocus = changedProp.currentValue;
+  }
```

Comment:
it looks like this is doing the exact same thing as the setter? why do we need so much complexity and duplication just to set a property value?